mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java

```diff
@@ -307,13 +307,41 @@ public void onBackPressed() {
                 return;
             }
             this.doubleBackToExitPressedOnce = true;
-            Toast.makeText(this, R.string.back_again, Toast.LENGTH_SHORT).show();
-            new Handler().postDelayed(new Runnable() {
-                @Override
-                public void run() {
-                    doubleBackToExitPressedOnce = false;
-                }
-            }, 2000);
+            FragmentManager fragmentManager = getSupportFragmentManager();
```

Comment:
Can you explain in your commit message, what is this change and why are we doing it in this way?